av1an-core/src/settings.rs

```diff
@@ -186,25 +186,26 @@ impl EncodeArgs {
     chunk: &Chunk,
     current_pass: u8,
     worker_id: usize,
+    tpl_crash_workaround: bool,
   ) -> Result<(), (EncoderCrash, u64)> {
     let fpf_file = Path::new(&chunk.temp)
       .join("split")
       .join(format!("{}_fpf", chunk.name()));
 
+    let mut video_params = self.video_params.clone();
+    if tpl_crash_workaround {
+      video_params.push("--enable-tpl-model=0".to_string());
```

Comment:
If the user explicitly enables `--enable-tpl-model=1`, does this conflict with that? I'm not familiar how the arg parsing in aomenc works.

Reply:
I just gave it a test and it looks like whatever is specified last will override earlier instances of the arg. Which is what we want here.